service-front/module/Application/test/Helpers/LocalisationHelperTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ApplicationTest\Helpers;
+
+use Application\Helpers\LocalisationHelper;
+use Application\Helpers\LpaFormHelper;
+use Laminas\Form\Annotation\AttributeBuilder;
+use Laminas\Form\FormInterface;
+use Laminas\Stdlib\Parameters;
+use PHPUnit\Framework\TestCase;
+use Application\Forms\LpaReferenceNumber;
+
+class LocalisationHelperTest extends TestCase
```

Comment:
This should have a test for `getInternationalSupportedDocuments`, since that's the only real publicly-exposed method